broker/broker.go

```diff
@@ -175,6 +181,12 @@ func (b *Broker) LastOperation(ctx context.Context, instanceID, operationData st
 	}
 
 	if state == NonExisting {
+		if operationData == OperationDeprovisioning {
```

Comment:
I'm not sure of the go conventions here, but I'd normally prefer to join these conditions in an `&&`, introduce an `else if state == NonExisting` and duplicate the end return value. This would reduce nesting and potentially make it clearer that there are separate code paths here. Thoughts?

Reply:
For me it's more clear with the current nesting level, so I suspect it's a personal preference.